utils/monitor/Makefile

```
@@ -0,0 +1,56 @@
+BUN := $(shell command -v bun 2> /dev/null)
+
+install:
+	@if [ ! $(BUN) ]; then \
+        echo "bun is not installed. Please install it first."; \
+        exit 1; \
+    fi
+	@echo "Installing monitor service..."
+	bun i
+	sudo cp monitor.service /lib/systemd/system/
+	sudo systemctl enable monitor
+.PHONY: install
+
+setup: install
+	sudo systemctl start monitor
+	sudo systemctl daemon-reload
```

Shouldn't this two lines be inverted?
```suggestion
	sudo systemctl daemon-reload
	sudo systemctl start monitor
```